simphony/io/h5_cuds.py

```diff
@@ -57,13 +57,31 @@ def open(cls, filename, mode="a", title=''):
             Title attribute of root node (only applies to a file which
               is being created)
 
+        filters : tables.Filter
+            Compression options used in the HDF5 file. If none is selected
```

Comment:
Does this only apply to new files?   If so, then we should add a statement stating that (like in the docstr for the `title` parameter).